NDB.Covid19/NDB.Covid19.Droid/Resources/layout/force_update.xml

```diff
@@ -5,6 +5,15 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <ImageView
+        android:id="@+id/force_update_logo"
+        android:layout_width="60dp"
+        android:layout_height="60dp"
+        android:layout_marginStart="@dimen/defaultHorizontalMargin"
+        android:layout_marginTop="@dimen/defaultVerticalMargin"
+        android:importantForAccessibility="yes"
```

Comment:
if it is important for accessibility why it does not have content description? in such case screen readers will focus this image but they will not describe it in any way.

Reply:
Added accessibility text on both platforms